sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/utils/AccessTokenUtil.java

```diff
@@ -74,6 +74,25 @@ public static String getAccessToken(String resource, String identity) {
         return result;
     }
 
+    /**
+     * Get an access token for a managed identity.
+     *
+     * @param resource the resource.
+     * @param identity the user-assigned identity (null if system-assigned)
+     * @return the authorization token.
+     */
+    public static OAuthToken getAccToken(String resource, String identity) {
+        OAuthToken result;
+
+        if (System.getenv("WEBSITE_SITE_NAME") != null
+            && !System.getenv("WEBSITE_SITE_NAME").isEmpty()) {
```

Comment:
Can we remove all methods that return `String`? just return `AccessToken`.

Reply:
I think Yes. But there is a public method that returns string. Not sure whether it is used outside our sight.